src/components/api.js

```diff
@@ -1,24 +1,63 @@
+/**
+ * @typedef ApiResponse
```

Comment:
It'd be cool if jsdoc could validate that all the responses will always return the typedef we've specified